rts/Lua/LuaVBOImpl.cpp

```diff
@@ -423,7 +493,7 @@ void LuaVBOImpl::Define(const int elementsCount, const sol::optional<sol::object
 		return false;
 	};
 
-	bool result;
```

Comment:
Linter complained about this undef, unsure how relevant but semmed harmless.

Reply:
Yep it's harmless. `LuaUtils::SolLuaError()` kills the Lua call, but linters are not aware of that.